hack/lib/tracing.bash

```diff
@@ -81,8 +81,8 @@ EOF
 function teardown_tracing {
   logger.warn 'Teardown tracing'
 
-  oc delete service    -n ${ZIPKIN_NAMESPACE} zipkin
-  oc delete deployment -n ${ZIPKIN_NAMESPACE} zipkin
+  oc delete service    -n "${ZIPKIN_NAMESPACE}" zipkin
+  oc delete deployment -n "${ZIPKIN_NAMESPACE}" zipkin
```

Comment:
What's this needed for? And related: Where is `ZIPKIN_NAMESPACE` defined? Couldn't find it in the repo.